src/System.Management.Automation/engine/Modules/TestModuleManifestCommand.cs

```diff
@@ -216,7 +216,7 @@ protected override void ProcessRecord()
                         {
                             foreach (ModuleSpecification moduleListModule in moduleListModules)
                             {
-                                var modules = GetModule(new[] { moduleListModule.Name }, true, true);
+                                var modules = GetModule(new[] { moduleListModule.Name }, false, true);
```

Comment:
this is a sync fix of the bug fix under windows bug 7980238.
so I would prefer to keep the code the way the original fix was